src/Component/navbar/Navbar.js

```diff
@@ -0,0 +1,28 @@
+import "./navbar.scss"
+import { Link } from "@mui/material";
+function Navbar() {
+  return (
+    <div className="navbar">
+      <div className="left">
+        <span><span className="left2">{"{"}</span>{"Finsweet"}</span>
+      </div>
+      <div className="right">
+        <Link to="/" style={{ textDecoration: "none"  ,"textAlign" :"center"}}>
+          <span>Home</span>
+        </Link>
+        <Link to="/" style={{ textDecoration: "none" }}>
+          <span>Blog</span>
+        </Link>
+        <Link to="/" style={{ textDecoration: "none" }}>
+          <span>About US</span>
+        </Link>
+        <Link to="/" style={{ textDecoration: "none" }}>
+          <span>Conact Us</span>
+        </Link>
+        <button type="Subscribe">Subscribe</button>
```

Comment:
The button design is not similar to the design 
<img width="677" alt="Screenshot 2022-11-20 at 8 38 56 PM" src="https://user-images.githubusercontent.com/8650435/202919813-8b28cbeb-920e-43b4-8a61-5f8343e42212.png"> 


but in the design 
<img width="588" alt="Screenshot 2022-11-20 at 8 39 44 PM" src="https://user-images.githubusercontent.com/8650435/202919854-861e97f7-7d71-4bcb-b837-b095e642b0a5.png">